practical/groovy/analyse_particles_for_another_user.groovy

```diff
@@ -62,17 +68,8 @@ import ij.plugin.frame.RoiManager
 import ij.measure.ResultsTable
 
 
-// Setup
-// =====
-
-// OMERO Server details
-HOST = "workshop.openmicroscopy.org"
-PORT = 4064
 group_id = -1
-//  parameters to edit
-dataset_id = 2331
-USERNAME = "username"
-PASSWORD = "password"
+
```

Comment:
This is confusing. The header of the script says that the agent doing the analysis is a user "analyst". But I cannot log in as "analyst". Instead, the script is just trying to harvest login details of user-x (the "target user") and run the analysis as user-x. But then, there is no element of "analysis for another"....